src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -37,25 +37,36 @@ public class EditCommand extends Command {
 
     public static final String COMMAND_WORD = "edit";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the contact identified "
-            + "by the index number used in the displayed contact list. "
-            + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: INDEX (must be a positive integer) "
+    public static final String MESSAGE_FUNCTION = COMMAND_WORD + ": Edits the details of the contact "
+            + "identified by the full name or the numerical index used in the displayed contact list. "
+            + "Existing values will be overwritten by the input values.";
+    public static final String MESSAGE_COMMAND_FORMAT = COMMAND_WORD + " "
+            + "INDEX (positive integer) OR FULL_NAME "
             + "[" + PREFIX_NAME + "NAME] "
-            + "[" + PREFIX_TELEGRAM_HANDLE + "telegramHandle] "
+            + "[" + PREFIX_TELEGRAM_HANDLE + "TELEGRAM_HANDLE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_STUDENT_STATUS + "STUDENT_STATUS] "
-            + "[" + PREFIX_ROLE + "ROLE]"
-            + "[" + PREFIX_NICKNAME + "NICKNAME]...\n"
-            + "Example: " + COMMAND_WORD + " 1 "
+            + "[" + PREFIX_ROLE + "ROLE[...]] "
+            + "[" + PREFIX_NICKNAME + "NICKNAME]"
+            + "\nformat in short: `" + COMMAND_WORD + " [INDEX OR FULL_NAME] [PREFIX] [new description]`";
```

Comment:
insert one for `MESSAGE_COMMAND_FORMAT_SHORT`. this should be the first one being used, the full command format will be used in help